platforms/chibios/vendors/RP/_pin_defs.h

```diff
@@ -35,3 +35,6 @@
 #define GP28 28U
 #define GP29 29U
 #define GP30 30U
+
+#define RP2040_PWM_CHANNEL_A 1
+#define RP2040_PWM_CHANNEL_B 2
```

Comment:
Probably shouldnt exist in pin defs, as its not coupled to GPIO

```suggestion
```

Reply:
The channels are coupled to the GPIOs every pin can only be one at a time, but yes it could probably live in `chibios_config.h` for the time being?

Reply:
After revisiting this I still think that `_pin_defs.h` while not optimal is the better option over e.g. `chibios_config.h`